sqlglot/dialects/doris.py

```diff
@@ -9,6 +9,21 @@
     time_format,
 )
 from sqlglot.dialects.mysql import MySQL
+from sqlglot.dialects.tsql import DATE_DELTA_INTERVAL
+
+
+def handle_date_trunc(self, expression: exp.DateTrunc) -> str:
+    expr = self.sql(expression, "this")
+    unit = expression.text("unit").lower()
+    if unit == "none":
```

Comment:
why would unit be the string "none"?

Reply:
When turnc has only one parameter, unit is none

Reply:
that is not true, unit can never be none. this code doesn't seem to be correct or you are probably misunderstanding how it's supposed to work

Reply:
![image](https://github.com/tobymao/sqlglot/assets/61498169/db1a9d89-bbd9-4038-8524-467e975d9e37)

Reply:
it's because the code you wrote in parser.py line 95 is wrong

Reply:
how should i get in touch with you

Reply:
tobikodata.com/slack